src/graphql/main/schema.graphql

```diff
@@ -1116,7 +1127,7 @@ type Transaction {
   """
   Price in USDCENT/SETTLEMENTUNIT at time of settlement.
   """
-  settlementPrice: Price!
+  settlementPrice: SettlementPrice!
```

Comment:
I don't think it's backward compatible. Are we using it already?

I'm not sure if there is an easy way on the backend to see what field are been used already. I think it was one of the benefits of Apollo Studio. would be relevant to know if we can create this capability back with honeycomb

Reply:
I didnt see validations in settlement price with `__typename`. also, this is a read only field.... we can merge it and test it in staging, I think it is not a problem

Reply:
true. we should check with the released version of the mobile app also.